app/models/evaluation.rb

```diff
@@ -37,4 +45,23 @@ def accepted
   def date_as_semester
     Semester.new(self.date)
   end
+
+  def default_values
+    self.point_request_status ||= NON_EXISTENT
+    self.entry_request_status ||= NON_EXISTENT
+    self.timestamp ||= Time.now
+    self.justification ||= ''
+    self.last_modification = Time.now
```

Comment:
If we return with self, then we could use these values.
But in this case that would be better if the method name were set_defult_values or something better.
i.e.: evaulation.set_default_values.timestamp